locales/es/translation.json

```diff
@@ -252,7 +263,9 @@
 	"MRN (optional)": "MRN (opcional)",
 	"My Data Sources": "Mis fuentes de datos",
 	"NAME": "NOMBRE",
+  "New Tidepool Account? Share Your Data with your healthcare team.": "Nueva cuenta? Comparta sus datos con su equipo de atención médica",
```

Comment:
This translation key should probably be added to the english translations file as well for completeness.